app/projects/models.py

```diff
@@ -50,6 +52,25 @@ class Project(TimestampedModelMixin, UUIDModelMixin, CreatedByModelMixin):
         through_fields=("project", "user"),
         related_name="project_memberships",
     )
+    normalised_domain = models.CharField(
+        max_length=256,
+        editable=False,
+    )
+
+    def clean(self):
+        # Create normalized version for uniqueness check
+        if self.domain:
+            normalised = self.domain.lower()
+            normalised = re.sub(r"^https://", "", normalised)
+            normalised = re.sub(r"^www\.", "", normalised)
```

Comment:
Why are we doing this? 

I understand the lowercase, and removing the `/` at the end, but why remove `https://` or `www`?

Reply:
I think removing the protocol makes sense, but then we should remove `https://` and `http://` and possibly other protocols.